tests/PHPStan/Rules/Variables/IssetRuleTest.php

```diff
@@ -277,10 +277,12 @@ public function testVariableCertaintyInIsset(): void
 				112,
 			],
 			[
-				'Variable $variableInFirstCase in isset() always exists and is not nullable.',
+				// could be Variable $variableInFirstCase in isset() always exists and is not nullable.
+				'Variable $variableInFirstCase in isset() is never defined.',
 				116,
 			],
 			[
+				// could be Variable $variableInSecondCase in isset() always exists and is not nullable.
 				'Variable $variableInSecondCase in isset() is never defined.',
```

Comment:
So this error already was incorrect. The other one is now also broken, because its condition is also used